### PR TITLE
Prepare google play

### PR DESCRIPTION
**Pull request by JcMinarro**
### 🎯 Goal
Prepare Demo App to be published on Google Play

### CODE REVIEW

**Review by skydoves — APPROVED** [review]
Looks good to me! 👍

File: .github/workflows/app-distribute.yml [hunk 1]
```diff
@@ -30,11 +30,11 @@ jobs:
...
+          path: dogfooding/build/outputs/apk/dogfooding/release/
       - name: Upload artifact to Firebase App Distribution
         uses: wzieba/Firebase-Distribution-Github-Action@v1.5.0
         with:
           appId: ${{secrets.FIREBASE_DOGFOODING_SAMPLE_APP_ID}}
           serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
           groups: stream-testers
-          file: dogfooding/build/outputs/apk/release/dogfooding-release.apk
\ No newline at end of file
+          file: dogfooding/build/outputs/apk/dogfooding/release/dogfooding-dogfooding-release.apk
```
Comment (kanat) [inline .github/workflows/app-distribute.yml:40]: > dogfooding-dogfooding-release.apk

It this an intentional doubling of the `dogfooding` word in the file name? :)

**Review by kanat — APPROVED** [review]
Overall, LGTM

File: .github/workflows/app-distribute.yml (hunk 1 above)
Comment (JcMinarro) [reply .github/workflows/app-distribute.yml:40]: As far as we are using `dogfooding`/`production` flavors and the project is named `dogfooding` as well, it is the result name used for the output apk.
We will change the project name in the future

**Pull request merged**